src/containers/PageBuilder/SectionBuilder/SectionBuilder.module.css

```diff
@@ -61,11 +78,19 @@
 .darkTheme h3,
 .darkTheme h4,
 .darkTheme h5,
-.darkTheme h6,
+.darkTheme h6 {
+  color: var(--matterColorLight);
+
+  &::selection {
+    background-color: cyan;
+    color: unset;
+  }
+}
+
 .darkTheme p,
 .darkTheme li,
 .darkTheme blockquote {
-  color: var(--matterColorLight);
+  color: rgba(255, 255, 255, 0.85);
```

Comment:
Is the opacity necessary? I.e. do you want the potential background image to be visible through text color?
But if you just want to use grey color, use grey color.